apothecary/formulas/uri/uri.sh

```diff
@@ -209,7 +209,6 @@ function build() {
         echo "Please stand by..."
         lipo -create libnetwork-uri-armv7.a \
                     libnetwork-uri-arm64.a \
-                    libnetwork-uri-i386.a \
```

Comment:
I think this is wrong, it's removing i386 from iOS lipo phase but it's not been removed from the build phase.